### ISSUE TITLE
While training using train_tts.py, the validation is weird.

### ISSUE THREAD

**Issue by kampores** [as-ideas/TransformerTTS#98]
When training using train_tts.py, I noticed that in tensorboard's images tab there is a yellow space to the right of the predicted_mel for validation.
![validation_mel](https://user-images.githubusercontent.com/5389649/114512965-a26b1c00-9c74-11eb-8e8b-4351958aa10b.png)
This is predicted_pitch for validation. There is no pitch in the yellow space.
![validation_pitch](https://user-images.githubusercontent.com/5389649/114515193-0e4e8400-9c77-11eb-9d58-f29f135803f5.png)
And the same as this yellow space position, there is also noise at the end of the validation wav in the audio tab of the tensorboard. Why is this happening? I'm not sure if this is my program setup mistake, what the author intended, or if it's a bug. explain it to me please.

**Comment by cfrancesco** [comment]
hi,
what you see/hear is the padding, is not removed from the validation mel spectrograms.

**Comment by kampores** [comment]
cfrancesco/ Oh I understood. Looking at this(https://stackoverflow.com/questions/62963127/when-using-padding-in-sequence-models-is-keras-validation-accuracy-valid-relia) question and answer, I found that padding doesn't affect validation accuracy. I think your answer means the same thing. Thank you.

**Comment by cfrancesco** [comment]
Hi,
what the author and participant refer to in the post does no directly apply to this case. In this case though the validation loss is masked (masked mse), so it will not be affected by padding.
